src/Protocol/Feed.php

```diff
@@ -503,7 +503,10 @@ public static function import($xml, array $importer = [], array $contact = [])
 				$items[] = $item;
 				break;
 			} elseif (!Item::isValid($item)) {
-				Logger::info('Feed is invalid', ['created' => $item['created'], 'uid' => $item['uid'], 'uri' => $item['uri']]);
+				Logger::info('Feed item is invalid', ['created' => $item['created'], 'uid' => $item['uid'], 'uri' => $item['uri']]);
+				continue;
+			} elseif (Item::tooOld($item)) {
```

Comment:
If this is only used here, please move the method to this file.

Reply:
I see it equal to the `isValid` function. So I think it does belong there. And possibly we want to use it for some other places in the future as well, like for example importing the outboxes of AP accounts.